src/plugins/formvalid/formvalid.js

```diff
@@ -106,12 +105,6 @@ var componentName = "wb-frmvld",
 						labels[ i ].innerHTML += " ";
 					}
 
-					// Remove the pattern attribute until it is safe to use with jQuery Validation
-					len = $pattern.length;
-					for ( i = 0; i !== len; i += 1 ) {
-						$pattern.eq( i ).removeAttr( "pattern" );
-					}
-
 					// Change form attributes and values that interfere with validation in IE7/8
```

Comment:
Can you remove the code block only applicable for IE8 and lower

Reply:
FWIW I have an upcoming PR whose goal will be to kill off old IE support. It'll remove that block.